internal/iam/repository_role_grant.go

```diff
@@ -593,39 +591,23 @@ func (r *Repository) grantsForUserGlobalResources(
 	if err := rows.Err(); err != nil {
 		return nil, errors.Wrap(ctx, err, op)
 	}
-
-	ret := make(perms.GrantTuples, 0)
-	for _, grant := range grants {
-		for _, canonicalGrant := range grant.canonicalGrants {
-			gt := perms.GrantTuple{
-				RoleId:            grant.roleId,
-				RoleScopeId:       grant.roleScopeId,
-				RoleParentScopeId: grant.roleParentScopeId,
-				GrantScopeId:      grant.roleScopeId, // use "global" for all global grants
-				Grant:             canonicalGrant,
-			}
-			ret = append(ret, gt)
-		}
-	}
-
-	return ret, nil
+	return grants, nil
 }
 
 // grantsForUserOrgResources returns the grants for the user for resources that can
 // only be org scoped.
 func (r *Repository) grantsForUserOrgResources(
 	ctx context.Context,
-	userId string,
+	userId,
+	scopeId string,
```

Comment:
nitpick: calling this request scope ID not to be confused with resource scope ID or role scope ID 
```suggestion
	reqScopeId string,
```